docs/release_notes.md

```diff
@@ -1,3 +1,327 @@
+
+ - Fix index usage in msiExtractTemplateMDFromBuf [#5010]
+
+ - Fix for missing O_TRUNC flag for puts [#5012]
+
+ - Replaced C header with C++ header [#5014]
+
+ - Add test for except PEP firing across federation [#5017]
+
+ - Fixes for imeta [#5021] [#5081] [#5101] [#5102] [#5111] [#5184] [#5185] [#5186] [#5518] [#5541]
+
+ - Fix for iscan limiting filenames to 256 characters [#5022]
+
+ - Fix for incorrect reuse of session properties [#5046]
+
+ - Fix for missing 'const noexcept' [#5048]
+
+ - Fix for the connection pool [#5053]
+
+ - Fix for duplicate error symbol in rodsErrorTable [#5056]
+
+ - Fix rebalance PEP documentation [#5062]
+
+ - Fix extraneous logging [#5064]
+
+ - Fix for incorrect reuse of logical path on reused connection [#5072]
+
+ - Fix for detecting empty hostnames [#5085]
+
+ - Fix for documentation of client_connection [#5092]
+
+ - Fixes for returning stale column output [#5099] [#5115]
+
+ - Fixes for irods::filesystem return good replica information [#5105] [#5116] [#5117] [#5119] 
+
+ - Fixes for filesystem function signatures [#5140]
+
+ - Fix for dereferencing std::optional [#5141]
+
+ - Fix for opening replicas of existing data objects [#5157]
+
+ - Fix for izonereport not including resc_id information [#5159] [#5170]
+
+ - Fix for rsDataObjRepl not returning errors from rsDataObjClose [#5179]
+
+ - Fixes for build system [#5198] [#5212] [#5213] [#5232] [#5233] [#5301] [#5311] [#5317] [#5354]
+
+ - Fix for iuserinfo use after free [#5214]
+
+ - Fixes for memory leaks [#5216] [#5299] [#5319] [#5337] [#5338] [#5339] [#5340] [#5341] [#5366] [#5367] [#5368] [#5369] [#5370] [#5371] [#5372] [#5373] [#5374]
+
+ - Document tcp_keepalive_time usage for long idle connections [#5218]
+
+ - Fix for leftover rulefile pid files [#5224]
+
+ - Fix for iinit [#5239]
+
+ - Fix for updating delayed rule last_exe_time when run by rodsuser [#5257]
+
+ - Fix for setup_irods.py to not start the server, leave it to systemd [#5275]
+
+ - Fix for POSIX semantics in streaming interface [#5315] [#5546]
+
+ - Add test to confirm reading past end of file is prohibited [#5352]
+
+ - Fix for setup_irods.py consistency [#5361]
+
+ - Fix for vault management during trim or remove [#5362]
+
+ - Fixes for irodsctl verbosity [#5383] [#5386]
+
+ - Fix for nanodbc packaging [#5389]
+
+ - Fix for extra logging from rule engine plugin framework [#5413]
+
+ - Quiet logs when interacting with nonexistent data objects [#5419] [#5444]
+
+ - Fix for unchecked input [#5420]
+
+ - Disable Xmsg tests [#5424]
+
+ - Fix for missing force flag when handling bundle files [#5426]
+
+ - Fix for unlimited rError stack size [#5427]
+
+ - Fix for API plugins triggering too much policy [#5437]
+
+ - Fix for irm with force flag [#5438]
+
+ - Fix for renaming local zone [#5445]
+
+ - Fix for msiRenameCollection to only rename collections [#5452]
+
+ - Fix for overwriting within special collections [#5457]
+
+ - Marked as wontfix or duplicate [#5472]
```

Comment:
Should this be moved into and/or near the resolved/invalid list?

Reply:
sure, combined.